systemtests/util_test.go

```diff
@@ -138,6 +138,7 @@ func (s *systemtestSuite) uploadIntent(tenantName, fileName string) (string, err
 }
 
 func (s *systemtestSuite) pullDebian() error {
+	log.Infof("Pulling debian:latest on all boxes")
```

Comment:
yeah this already existed before, I just omitted the log message when I changed it to use the pure `vagrantssh` function